src/sentry/monitors/endpoints/monitor_ingest_checkin_details.py

```diff
@@ -90,7 +90,9 @@ def put(
             params["duration"] = duration
 
         monitor_environment = MonitorEnvironment.objects.ensure_environment(
-            project, monitor, params.get("environment")
+            project,
+            monitor,
+            result.get("environment"),
```

Comment:
style change. lint won't let me undo

Reply:
It's because of the trailing comma on `result.get("environment"),`